Added note to not use reserved tags as host level tags (#11006)
DataDog/documentation: content/en/tracing/guide/setting_primary_tags_to_scope.md

* Only organization administrators have access to this page.
* Changes may take up to two hours to be reflected in the UI.

> * There are some tags that the tracer always adds to spans: `resource`, `name`, and `service`. To avoid potentially confusing UI behavior never add these as host level tags. 

Reviewer: ```suggestion
* The tracer always adds `resource`, `name`, and `service` tags to spans. Datadog recommends never adding these as host level tags to avoid confusion.
```
